src/main/java/uk/gov/register/db/ItemQueryDAO.java

```diff
@@ -29,7 +29,7 @@
     @FetchSize(262144)
     ResultIterator<Item> getIterator();
 
-    @SqlQuery("select * from item where exists(select 1 from entry where item.sha256hex = entry.sha256hex and entry_number between :startEntryNo and :endEntryNo) order by sha256hex")
+    @SqlQuery("select * from item where exists(select 1 from entry where item.sha256hex = entry.sha256hex and entry_number > :startEntryNo and entry_number <= :endEntryNo) order by sha256hex")
```

Comment:
Likewise if we call /download-rsf/1/2, we only want to get item #2, which this change ensures.